packages/core/src/index.ts

```diff
@@ -401,21 +405,5 @@ export const applyNetworkInterceptors = (
   return createTruePromise();
 };
 
-const createFalsePromise = (): Promise<boolean> => {
```

Comment:
I'd leave these in place for the purposes of this PR, can revisit refactoring or replacing with `Promise.resolve` where they get used separately

Reply:
sure